mongodb/Acornfile

```diff
@@ -0,0 +1,331 @@
+args: {
+    // Specify whether MongoDB instance is a standalone or replica set
+    isReplicaSet: false
+    // Custom user to be created during the initialization
+    dbUserName: ""
+    // Custom database to be created during the initialization
+    dbName: "acorn"
+    replicas: 1
+    // Enable diagnostic mode
+    diagnosticMode: false
+    // Enable authentication
+    authEnabled: true
+    // Run arbiter node
+    // Mongodb additional command line flags
+    extraFlags: ""
+    arbiterEnabled: false
+    // Arbiter additional command line flags
+    arbiterExtraFlargs: ""
+    // Run hidden node
+    hiddenEnabled: false
+    hiddenReplicas: 1
+    // Hidden node additional command line flags
+    hiddenExtraFlargs: ""
+    // App installed namespace. NOTE(joseb): before fieldRef is supported,
+    appNamespace: "mongo"
+}
+profiles: {
+    prod: {
+        isReplicaSet: true
+        arbiterEnabled: true
+        hiddenEnabled: true
+        replicas: 3
+    }
+}
+    
+if !args.isReplicaSet {
+    containers: {
+        "mongodb-0": {
+            image: "bitnami/mongodb:6.0.1-debian-11-r1"

```

Comment:
Can we use https://hub.docker.com/_/mongo image?

Reply:
well, I tried the image maintained by docker community but it is too vanila to operate. I attempted to make utility scripts one by one and migrate scripts from bitnmi images and asked myself why can't use bitnami image directly. 

Btw, If that is mandatory rule, i will use official one, so plz let me know

Reply:
bitnami removed